### PR TITLE
Feature/login

### LINKED ISSUE #12: Component | Login Page
- [ ] structure, plan and style login page

### LINKED ISSUE #14: User Story | Login
- [ ] As a user, I should be able to log into my account when the application loads

### PR DESCRIPTION
**Pull request by Wperricone**
## What was changed:
- A login page was added, as well as error handling for a bad login, and a logout functionality

## Explain the reason for changes:
- This functionality allows a user to login, see their profile, which includes their items they have listed. Additionally, a user can logout so they are no longer logged in. 

## Link to issue:
- close #14 close #12

### CODE REVIEW

**Comment by JoeRupp** [comment]
Awesome work with the error handling. We'll need to build out some tests for this page later.

File: src/Components/Login.js [hunk 1]
```diff
@@ -1,16 +1,123 @@
...
+				description
+				available
+				category
+				status
+				amount
+			}
+    }
+  }
+`;
+
```
Comment (stephanie-roe) [inline src/Components/Login.js:27]: moving this query and restructuring it seems a lot more streamlined- great choice!

**Comment by Abumpus1** [comment]
Glad we could involve the back end on this!

File: src/Components/Login.js [hunk 2]
```diff
@@ -1,16 +1,123 @@
...
-                <input type="text" placeholder="email"></input>
-                <input type="text" placeholder="password"></input>
-                <button>login</button>
-            </div>
-        </section>
+        <LoginPageSection className="login-page">
+            <LoginBox className="login-modal">
+							{badLogin && <LoginError>Invalid email or password. Please try again.
+							</LoginError>
+							}
```
Comment (stephanie-roe) [inline src/Components/Login.js:58]: great use of error handling here!

**Review by stephanie-roe — APPROVED** [review]
Great job! I like how you added error handling and remembered the log out functionality.

**Pull request merged**